clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -97,6 +97,9 @@ The improvements are...
 Improvements to clang-tidy
 --------------------------
 
+- Improved :program:`run-clang-tidy.py` script. Added argument `-source-filter`
+  to filter out source files from the compilation database.
```

Comment:
I read this as "the option 'source-filter' is used to filter out source files". But in reality it's doing the opposite: it will filter **in** those source files (just like header-filter).

Maybe we can write something like "with the same semantics of header-filter, but for source files"?

Reply:
Ok, I clarified it and mentioned `-header-filter`.